src/main/java/me/marin/lockout/mixin/client/InputMixin.java

```diff
@@ -18,14 +19,9 @@ public void tick(boolean slowDown, float slowDownFactor, CallbackInfo ci) {
 
         KeyboardInput input = (KeyboardInput) (Object) this;
         if (!LockoutClient.lockout.hasStarted()) {
-            input.pressingForward = false;
-            input.pressingBack = false;
-            input.pressingLeft = false;
-            input.pressingRight = false;
+            input.playerInput = PlayerInput.DEFAULT;
```

Comment:
sneaking now gets set to false
i think sneaking should be allowed during wait time, but it's whatever